test.sh

```diff
@@ -93,6 +93,13 @@ check_print()
     fi
 }
 
+fastmath()
```

Comment:
I think this functionshould be called `check_fastmath`

Reply:
Makes sense! Thanks for pointing that out.